backend/src/federation/client/Client_1_0.ts

```diff
@@ -0,0 +1,46 @@
+import { FederatedCommunity as DbFederatedCommunity } from '@entity/FederatedCommunity'
+import { GraphQLClient } from 'graphql-request'
+
+import { getPublicKey } from '@/federation/query/getPublicKey'
```

Comment:
This is analog to all our other graphql query implementation. We always have the query itself separated in a folder outside of the actuall calling implementation. This allows to reuse it and have all graphql queries in one place wherever they are called. Often it is the case that a test reuses such a query.

I agree tho that this is one step in the future which i could have skipped since there is no other query yet.

Reply:
If you want I can move this inside the function again.

Reply:
no that's ok for me.